plugins/inputs/logparser/logparser_test.go

```diff
@@ -14,11 +15,13 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const pathSeparatorString = string(os.PathSeparator)
+
 func TestStartNoParsers(t *testing.T) {
 	logparser := &LogParserPlugin{
 		Log:           testutil.Logger{},
 		FromBeginning: true,
-		Files:         []string{"testdata/*.log"},
+		Files:         []string{"testdata" + pathSeparatorString + "*.log"},
```

Comment:
Maybe `filepath.Join()`? Same for the instances below...

Reply:
Sure! Done.

Reply:
Mark as resolved.